students/mimalvarez_pintor/lesson03/list_lab.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Apr 26 20:35:32 2020
+
+@author: miriam
+"""
+
+# Series 1
+L = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(L)
+F = input("Please type in a fruit: ")
+L.append(F)
+print(L)
```

Comment:
make sure you to follow pep8 style guidelines for python:
* use lower case snake_case for var names
* use more user friendly variable names


so in this case it would be `fruit_list` and `fruit` for example